docs/src/surfaces/actionBar/actionBarSample.jsx

```diff
@@ -0,0 +1,80 @@
+import { ActionBar } from 'react-cm-ui';
+import makeStyles from 'react-cm-ui/styles/makeStyles';
+import React from 'react';
+
+const useStyles = makeStyles(() => ({
+    root: {
+        position: 'static',
+    },
+}));
+
+function ActionBarSample() {
+    const classes = useStyles();
+
+    return (
+        <ActionBar
+            classes={{
+                root: classes.root,
+            }}
+            justifyContent="flex-end"
+            columns={[
+                {
+                    button: {
+                        color: 'alternate',
+                        iconType: 'envelope',
+                        title: 'SMS',
```

Comment:
Should this be `'Email'`? (since SMS is the next button with `comment-lines` icon type)

Reply:
Fixed.

Reply:
Looks like we removed diffs in this file? 😕

Reply:
Correct. Many of the changes that happened should have been made. This one got some reverted changes on it.

Reply:
Cool.  Well, resolving this.